api/envoy/config/core/v3/protocol.proto

```diff
@@ -24,6 +24,12 @@ message TcpProtocolOptions {
       "envoy.api.v2.core.TcpProtocolOptions";
 }
 
+message QuicProtocolOptions {
```

Comment:
Maybe doc/comment about how these are protocol options shared between both downstream and upstream?